Casks/j/jamovi.rb

```diff
@@ -1,15 +1,18 @@
 cask "jamovi" do
-  version "2.3.28.0"
-  sha256 "4beeca531e3d816fa38bcb363a92967beba316f5ffbd4f1867da87b4edc79161"
+  arch arm: "arm64", intel: "x64"
 
-  url "https://www.jamovi.org/downloads/jamovi-#{version}-macos.dmg"
+  version "2.6.44.0"
+  sha256 arm:   "30ac4a36515b937d1c83eb5034d51a06695c079799364b341340c5c51e83ce2f",
+         intel: "9301617a03a0e0edcb9a5113649820ff04161cd367ff6c5135af75ae3fa3964b"
+
+  url "https://www.jamovi.org/downloads/jamovi-#{version}-macos-#{arch}.dmg"
   name "jamovi"
   desc "Statistical software"
   homepage "https://www.jamovi.org/"
 
   livecheck do
-    url "https://www.jamovi.org/download.html"
-    regex(/href=.*?jamovi[._-]v?(\d+(?:\.\d+)+)[._-]macos\.dmg/i)
+    url "https://dl.jamovi.org/"
```

Comment:
```suggestion
    url "https://www.jamovi.org/download.html"
```

If upstream provides both a download page and a directory listing page where the assets are found, it's preferable to check the download page as long as there aren't any issues with it (e.g., it's not reliably kept up to date, etc.). In this case, the download page links to the latest files, so we should continue checking the download page.

The rationale behind this is that upstream may upload files before the version is officially released and they may feel comfortable deleting/replacing the files, etc. anytime before they publish the release. If we check the directory listing page, livecheck may surface a new version before it's officially released and that could create issues (both for us and any users who install the new version). Checking the download page ensures that livecheck is only surfacing versions that upstream has published as released.

Reply:
Makes sense to me, thanks!